app/uk/gov/hmrc/alcoholdutyreturns/models/returns/AdrReturnSubmission.scala

```diff
@@ -224,7 +378,24 @@ case class AdrSpiritsProduced(
   otherMaltedGrainType: Option[String],
   ingredientsVolumes: AdrSpiritsIngredientsVolumes,
   otherIngredient: Option[AdrOtherIngredient]
-)
+) {
```

Comment:
Agreed, the other place they are mentioned are in general grains volumes. We have potential to later rework the structure as it's frontend <-> backend only, but I'm not sure there is going to be a perfect way, and flattening it makes it worse because of all the fields.